Pareto/AppInfo.swift

```diff
@@ -77,14 +76,10 @@ enum AppInfo {
         #if SETAPP_ENABLED
             source += "-setapp"
         #else
-            if Licensed {
-                if Defaults[.teamID].isEmpty {
-                    source += "-personal"
-                } else {
-                    source += "-team"
-                }
+            if Defaults[.teamID].isEmpty {
+                source += "-personal"
             } else {
-                source += "-opensource"
+                source += "-team"
             }
         #endif
```

Comment:
This will break stats. It' needs to stay opensource and team

Reply:
👍 I guess `ReportingRoles` should also be `team` and `opensource` then?